recipe/meta.yaml

```diff
@@ -35,12 +35,15 @@ outputs:
     build:
       script: python -m pip install . --ignore-installed -vv
     requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - cython                                 # [build_platform != target_platform]
+        - {{ compiler('cxx') }}
```

Comment:
@xhochy, got it, thanks! Could you also please tell me why this is necessary to build an `osx_arm64` package on `osx_64`? And what is the actual difference between `osx_arm64` and `osx_64`?

Reply:
> why this is necessary to build an osx_arm64 package on osx_64? 

We don't have `osx-arm64` runners in CI, so we need to cross-compile.

>  And what is the actual difference between osx_arm64 and osx_64?

One is an ARM CPU, the other one an Intel CPU. So they have completely different instruction sets.

Reply:
Thank you for the responses.

Reply:
@xhochy, btw, will support for Apple Silicon processors be added for all conda-forge packages? Why am I asking because we also support [modin-feedstock](https://github.com/conda-forge/modin-feedstock) but a related PR hasn't been opened yet there.

Reply:
Only for those where someone requested the addition via an entry in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/osx_arm64.txt

Reply:
I see a opened PR in modin-feedstock now. Could you help us with it there?